indico/modules/events/timetable/templates/_draft_mode_warning.html

```diff
@@ -0,0 +1,26 @@
+{% macro render_draft_mode_warning(event) %}
+
+<div id="draft-mode-warning" class="action-box warning draft-mode">
+    <div class="section">
+        <div class="icon icon-edit"></div>
+        <div class="text">
+            <div class="label">{% trans %}Contributions are in draft mode{% endtrans %}</div>
+            {% trans %}
+                While in draft mode, regular users cannot see the contributions and timetable.
+            {% endtrans %}
+        </div>
+        <div class="toolbar">
+            <div class="group">
+                <button id="draft-mode-warning-button" class="i-button" data-event-id="{{ event.id }}">
+                </button>
```

Comment:
I think it'd be better to just put an empty span with the ID and data attribute here and then let the JSX render the IButton